spec.html

```diff
@@ -30838,9 +30839,9 @@ <h1>
         </emu-clause>
       </emu-clause>
 
-      <emu-clause id="sec-symbol.prototype-@@toprimitive">
-        <h1>Symbol.prototype [ @@toPrimitive ] ( _hint_ )</h1>
-        <p>This method is called by ECMAScript language operators to convert a Symbol object to a primitive value.</p>
+      <emu-clause oldids="sec-symbol.prototype-@@toprimitive" id="sec-symbol.prototype-%symbol.toprimitive%">
+        <h1>Symbol.prototype [ %Symbol.toPrimitive% ] ( _hint_ )</h1>
+        <p>This function is called by ECMAScript language operators to convert a Symbol object to a primitive value.</p>
```

Comment:
Accidental revert of PR #2592?
```suggestion
        <p>This method is called by ECMAScript language operators to convert a Symbol object to a primitive value.</p>
```